documentation/deliberately_vulnerable_flask_app/README.md

```diff
@@ -3,6 +3,33 @@ This is a trivially vulnerable Flask application, meant as a playground for
 testing Pysa
 
 # Setup
-1. `pip install pyre-check-nightly`
-2. `pyre init`
-3. `pyre analyze --no-verify`
+1. Setup a virtual environment (if you haven't already):
+```bash
+python -m venv ../.env
+source ../.env/bin/activate
```

Comment:
Any reason we recommend to make this a hidden directory?
As a python user, I like to see where are my virtual environments.

Reply:
None whatsoever! I will make it non-hidden